utoipa-gen/src/lib.rs

```diff
@@ -534,17 +537,17 @@ pub fn derive_component(input: TokenStream) -> TokenStream {
 /// **For example:**
 ///
 /// ```text
-/// ("id" = String, path, deprecated, description = "Pet database id"),
-/// ("id", path, deprecated, description = "Pet database id"),
-/// (
-///     "value" = inline(Option<[String]>),
-///     query,
-///     description = "Value description",
-///     style = Form,
-///     allow_reserved,
-///     deprecated,
-///     explode,
-///     example = json!(["Value"])
+/// params(MyParameters)
+/// ```
+///
+/// Note that `MyParameters` can also be used in combination with the [tuples
+/// representation](#tuples) or other structs. **For example:**
+///
+/// ```text
+/// params(
+///     MyParameters1,
+///     MyParameters2,
+///     ("id" = String, path, deprecated, description = "Pet database id"),
```

Comment:
I guess there are still few places in docs where `path` should be represented as `Path` probably good idea to go through them. I can also check them as well and fix what I find.  
```suggestion
///     ("id" = String, Path, deprecated, description = "Pet database id"),
```